addons/gametime/gametime.lua

```diff
@@ -16,7 +16,7 @@
 -- THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 -- ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
 -- WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
--- DISCLAIMED. IN NO EVENT SHALL <your name> BE LIABLE FOR ANY
+-- DISCLAIMED. IN NO EVENT SHALL OMNYS BE LIABLE FOR ANY
```

Comment:
Just for the record, your name does not need to be in capital letters, but either works!